data-manager/pom.xml

```diff
@@ -31,6 +31,10 @@
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy-sql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-templates</artifactId>
+        </dependency>
```

Comment:
you should add this dependency with its version in the parent `<dependencyManagement>`